bindings/cpp/logger.cpp

```diff
@@ -137,13 +137,12 @@ void dnet_node_set_trace_id(dnet_logger *logger, uint64_t trace_id, int tracebit
 	if (scoped_count >= DNET_SCOPED_LIMIT) {
 		dnet_log_only_log(logger, DNET_LOG_ERROR,
 			"logic error: you must not call dnet_node_set_trace_id twice, dnet_node_unset_trace_id call missed");
-		++scoped_count;
```

Comment:
All calls of `dnet_node_set_trace_id()` are followed by calls of `dnet_node_unset_trace_id()`. So if it doesn't increment `scoped_count` here the following call of `dnet_node_unset_trace_id()` will unset someone's, but not own, trace_id.